### DISCUSSION TITLE
Albumart for non stop Streaming Stations (correlate streamer metadata with existing tracks in Media folders)

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#7223] [Q&A]
Hello! 

I was wondering if this could be possible... now or maybe as a future feature: 
If the streamer metadata title is also present in media folder, the player will show that specific media albumart, instead of the default albumart.

**Comment by BusterNeece** [comment]
We do have something somewhat similar to what you're asking for, which is that if there's a file named `cover.jpg` or something similar, we process that as the default album art for every media file in that folder.

**Reply by unudelaradio** [reply]
What i need, more exactly, is a method to provide custom covers/albumarts for my streaming station. When a station is most of the time in live streaming, the artwork is static: the player only shows the default albumart for that station.

So i was thinking it would be great if the metadata sent by the streamer could be assosiated, somehow, with the same track already present in Media Folders (and get and show it's albumart in the player, if streamer plays that track). 

Or maybe creating a new place, a custom folder eg `.streamer_covers` in which i can upload my artworks via SFTP, with the exact names as streamer's music titles.

Example:
Streamer Metadata: `CYRIL - Stumblin' In`
File in the `.streamer_covers` folder: `CYRIL - Stumblin' In.jpeg`

Why am i suggesting this? Because, unfortunately, last.fm is not accurate, most of the times.
